pints/_optimisers/__init__.py

```diff
@@ -186,9 +186,9 @@ def __init__(
                 ' optimise.')
 
         # Store function
-        if isinstance(function, pints.LogLikelihood):
+        if isinstance(function, pints.LogPDF):
             # Likelihood function given? Then wrap an inverter around it
-            self._function = pints.LogLikelihoodBasedError(function)
+            self._function = pints.ProbabilityBasedError(function)
```

Comment:
Can we have some kind of stdout or print statement saying _explicitly_ that we are now maximising the input `function`? And vise versa (say now minimising the function when it is not pints.LogPDF). Either here or later when it actually starts the optimisation (or least under `verbose` option, if we still have it here).

We have "Optimisation" as the name, when debugging, something people might be not sure if it is their function's problem (i.e. the objective function is doing weird things) or they just made a silly mistake that they are minimising something that they should maximise.

I found this would be helpful particularly when the program is trying to do something smart for me -- because something I might try to be smart that undo what the program nicely did for me... So might happen to other users as well.

Reply:
Good point! Have added it to the documentation since you used it, but will also add a line to the verbose output!